test.ts

```diff
@@ -16,21 +14,12 @@ import "fs/path/relative_test.ts";
 import "fs/path/resolve_test.ts";
 import "fs/path/zero_length_strings_test.ts";
 import "io/bufio_test.ts";
+import "io/ioutil_test.ts";
```

Comment:
Aside: I wasn't sure if ioutil should remain here,  it's a badly named file atm.
If ioutil is useful outside of ws we should be a little more descriptive...

Reply:
I agree the name isn't great - but it comes from go - so at least there's that president : https://golang.org/pkg/io/ioutil/
I think it's useful outside of WS